crypto/curve25519/internal.h

```diff
@@ -28,6 +28,27 @@ void x25519_NEON(uint8_t out[32], const uint8_t scalar[32],
                      const uint8_t point[32]);
 #endif
 
+#if !defined(OPENSSL_NO_ASM) && !defined(OPENSSL_SMALL) && \
```

Comment:
When I merged 43f88915f97d1f7df489b4b706afdfa556ba2050 I should have merged this, but I had wrongly thought it was going to be unneeded, so I didn't. Do it now.